src/test/java/hudson/plugins/jira/JiraCreateReleaseNotesTest.java

```diff
@@ -62,8 +63,7 @@ public class JiraCreateReleaseNotesTest {
     @Mock
     JiraSite site;
 
-    @Mock
-    private PrintWriter printWriter;
+    private final PrintWriter printWriter = new PrintWriter(OutputStream.nullOutputStream());
```

Comment:
[nit] The only real reason to make this a field in the first place was to use the `@Mock` annotation on it, so now that `@Mock` is being deleted this could be inlined and would be arguably more readable that way. But I do not feel strongly either way.